CHANGES.rst

```diff
@@ -1,3 +1,8 @@
+unreleased
+==========
+
+- Add `jinja2.i18n_extension` configuration setting.
```

Comment:
reStructuredText requires double backticks for inline literals, whereas Markdown uses single.

```suggestion
- Add ``jinja2.i18n_extension`` configuration setting.
```